Orange/tests/test_base.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from Orange.base import SklLearner
+
+
+class TestSklLearner(unittest.TestCase):
+    def test_supports_weights(self):
+        class DummySklLearner:
+            def fit(self, X, y, sample_weight=None):
+                pass
+
+        class DummyLearner(SklLearner):
+            __wraps__ = DummySklLearner
+
+        self.assertTrue(DummyLearner().supports_weights)
```

Comment:
Could we base this test on one real sklearn's learner, so it will fail when, if, they decide to change the `sample_weight` argument name.